pkg/printers/internalversion/import_known_versions.go

```diff
@@ -19,6 +19,7 @@ package internalversion
 // These imports are the API groups the client will support.
 // TODO: Remove these manual install once we don't need legacy scheme in get comman
 import (
+	// Import side effects of the following packages
```

Comment:
I think it would be enough to bump the comments above `import` here if golint complains about it.

Reply:
It's not the comments above `import` but the imports themselves. Apparently, golint expects blank imports to be preceded with a comment justifying them if not inside a `main` or `test` package.

Reply:
I meant to say that we should move the comments on line 19-20 here since they seem to explain why we have these imports (for installing API groups)

Reply:
Oh, that makes sense. 
Fixed.